### ISSUE TITLE
Quorum & acceptance calculation for article BUIPs needs to be different

### ISSUE THREAD

**Issue by torusJKL** [BitcoinUnlimited/BitcoinUnlimitedVotingWebService#14]
For BUIP81 we see that 13 members have voted and the voting system shows this as a quorum of 25% of the members.
In addition the vote is accepted with a 51% majority.

But for Article changes the following applies:

> This document can be modified via a greater than 66% 
> majority vote on a BUIP with at least 75% of the 
> members voting.

The voting system should be able to distinguish between these different types of BUIPs and calculate the quorum and acceptance accordingly.

**Comment by sickpig** [comment]
@torusJKL you are right indeed.

**Comment by awemany** [comment]
Yes, agreed, I need to implement voting for article change BUIPs. Of course, 'the code is not the law' here (c.f. Bitcoin Core) and the articles apply and not my bogus calculation :)

Thanks for reporting!

Given that we also have out of band voting still (on bitco.in), I likely need to add a comment that can be set by the vote master to indicate an override of the vote due to facts outside of the system. Long term, of course, I like this system to become definitive, because otherwise it is pointless.
